src/Illuminate/Routing/Router.php

```diff
@@ -613,7 +613,7 @@ protected function mergeGroupAttributesIntoRoute($route)
      * @param  array  $action
      * @return bool
      */
-    protected function actionReferencesController($action)
+    public function actionReferencesController($action)
```

Comment:
protected methods can be overwritten by a parent class, and modifying the visibility to public will cause a fatal error